src/libs/EmojiTrie.ts

```diff
@@ -95,9 +96,22 @@ function createTrie(lang: SupportedLanguage = CONST.LOCALES.DEFAULT): Trie<Emoji
     return trie;
 }
 
-const emojiTrie: EmojiTrie = supportedLanguages.reduce((prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
+// const emojiTrie: EmojiTrie = supportedLanguages.reduce((prev, cur) => ({...prev, [cur]: createTrie(cur)}), {});
+const emojiTrie: EmojiTrie = {
+    en: undefined,
+    es: undefined,
+};
```

Comment:
Instead of manually creating and maintaining this variable can we consider implementing something like:

https://github.com/callstack-internal/Expensify-App/blob/f903144c44c252e676670ad8d8251f992219cfd6/src/libs/Localize/index.ts#L68-L73

Reply:
That approach I think will create all the tries for "en" and "es" and we want to create them on demand, that's why the idea is to create them manually when it is needed

Reply:
good point, I added the change so now we create it based on `supportedLanguages`  variable, thanks!